requirements/edx/github.in

```diff
@@ -74,3 +74,6 @@ git+https://github.com/edx/django-ratelimit-backend.git@v2.0.1a5#egg=django-rate
 
 git+https://github.com/open-craft/xblock-poll@922cd36fb1c3cfe00b4ce03b19a13185d136447d#egg=xblock-poll==1.10.2
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.3.4#egg=xblock-drag-and-drop-v2==2.3.4
+
+# RG
+-e git+https://github.com/raccoongang/edx-proctoring@oex_proctoring#egg=edx-proctoring
```

Comment:
@UvgenGen please add empty line in the end of the file

Reply:
done